source/programming/markdown.rst

```diff
@@ -33,3 +40,6 @@ VS Code 编辑器的 Markdown 语言扩展支持实时预览 Markdown 文档。
 - https://daringfireball.net/projects/markdown/dingus
 
 推荐使用 `pandoc <https://pandoc.org/>`__ 将 Markdown 文档转成其他格式，如 PDF、LaTeX。
+
+推荐使用浏览器插件 `Markdown Here <https://markdown-here.com/>`__\ ，以便使用 Markdown
+语法写优美的邮件。
```

Comment:
Are you sure it still works? It's not updated for more than 2 years (https://github.com/adam-p/markdown-here).

Reply:
I can still use it in Google-Chrome in CentOS7.

Reply:
It's no longer maintained. I don't think we should recommend it.

Reply:
Removed.